docs/source/algorithms/SaveNexusGeometry-v1.rst

```diff
@@ -0,0 +1,46 @@
+
```

Comment:
It's very important that this documentation page is completed and describes the new algorithm well.

Advice:
1. You should provide a link to the Nexus Format manual and explain why this algorithm exists.
1. You should mention some more information about the optional property and explain exactly what it does in the output nexus file.
1. You need to explain the caveat that Shapes are NOT saved in the present version!
1. You will need to provide a doc test. You can use `CreateSampleWorkspace` to create a nice test scenario, or use `LoadEmptyInstrument` to create a more specific customised instrument if you wish.
1. Related algorithms are `LoadInstrument` (which has a nexus route).

You will also need to update the release notes.

@TakudzwaMakoni please work with @DanNixon if you have any questions on this.